loom/graphics/gfxScript.cpp

```diff
@@ -357,6 +358,34 @@ static int registerLoomGraphics(lua_State *L)
        .addProperty("path", &TextureInfo::getTexturePath)
        .endClass()
 
+       .beginClass<Shader>("Shader")
+       .addConstructor<void(*)(void)>()
+       .addVarAccessor("onBind", &Shader::getonBindDelegate)
+       .addVarAccessor("MVP", &Shader::getMVP)
+       .addVarAccessor("textureId", &Shader::getTextureId)
+       .addMethod("load", &Shader::load)
+       .addMethod("loadFromAssets", &Shader::loadFromAssets)
+       .addMethod("bind", &Shader::bind)
```

Comment:
Oh, I accidentally left that there when I thought I could make it inheritable... Coudn't assign an inherited class to .shader property :(